### ISSUE TITLE
django v4 complains about a missing "cuser" migration

### ISSUE THREAD

**Issue by macolo** [tmm/django-username-email#24]
- django v4 complains about a missing "cuser" migration
- manage.py makemigrations creates the following "" migration.
- I suspect that this migration must be added to cuser source code for django v4 compatibility

```

root@7a55490bd5f3:/app# ./manage.py makemigrations
Migrations for 'cuser':
  /usr/local/lib/python3.9/site-packages/cuser/migrations/0004_alter_cuser_id.py
    - Alter field id on cuser

root@7a55490bd5f3:/app# cat /usr/local/lib/python3.9/site-packages/cuser/migrations/0004_alter_cuser_id.py
# Generated by Django 4.0 on 2021-12-17 09:39

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('cuser', '0003_alter_user_first_name_max_length'),
    ]

    operations = [
        migrations.AlterField(
            model_name='cuser',
            name='id',
            field=models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID'),
        ),
    ]


```

**Comment by ataylor32** [comment]
Thanks for the report! I'll look into this at some point (probably within the next couple of months).

**Comment by ataylor32** [comment]
Fixed in 69ad65c4d1b0602ba4f2909c233c8b47169a09ad. I will need to release a new version, but I don't think this repo's "automatically publish to PyPI" feature is working right now. @tmm can you confirm when you have some time? Previously, I could change the version number in setup.py, commit it, tag the commit, and push, which would cause the new version to automatically get published to PyPI.

**Comment by tmm** [comment]
@ataylor32 looks like the CI setup with Travis stopped working because they no longer has a free plan. Seems like the best option would be to switch to GitHub Actions.

I don't use Django (and Python) that much anymore so would be happy to transfer this project (GitHub repo, pypi package, etc.) to you so it's easier to maintain.

**Comment by ataylor32** [comment]
@tmm Thank you, I would be fine with that.

**Comment by tmm** [comment]
@ataylor32 great! what's your pypi username?

**Comment by ataylor32** [comment]
@tmm ataylor32

**Comment by tmm** [comment]
Just added you as an owner to the pypi package. Looks like you need to delete a fork so I can transfer the repo to you.

<img width="373" alt="image" src="https://user-images.githubusercontent.com/6759464/172453544-e7352c72-52de-4dc0-816f-ca4a68c60c1f.png">

**Comment by ataylor32** [comment]
@tmm Thank you. I deleted my fork just now.

**Comment by tmm** [comment]
Transfer request submitted!

**Comment by tmm** [comment]
Thanks for maintaining for all these years! If you find yourself in Brooklyn, NYC, I owe you a drink of your choice :)

**Comment by ataylor32** [comment]
@tmm You're welcome, and thank you!

**Comment by ataylor32** [comment]
@macolo Fixed in version 2.5.3
